### PR TITLE
data:Kgkunal

### PR DESCRIPTION
**Pull request by Kgkunal**
## Check List (Check all the applicable boxes) 

- [ ] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [ ] This PR does not contain plagiarized content.
- [ ] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5495"></a>

### CODE REVIEW

File: data/Kgkunal.json [hunk 1]
```diff
@@ -0,0 +1,18 @@
+{
+  "name": "Kunal Gavhane",
+  "type": "personal",
+  "bio": "👨‍💻 Software Developer | Tech-Writer | Open Source",
+  "avatar":"https://github.com/Kgkunal.png",
```
Comment (vedprakashnautiyal) [inline data/Kgkunal.json:5]:
```suggestion

```

Comment (vedprakashnautiyal) [reply data/Kgkunal.json:5]: You don't have to use avatar....

**Review by vedprakashnautiyal — CHANGES_REQUESTED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by Kgkunal** [comment]
> looks good

Thanks alot  🤗

**Pull request merged**